WebAPI.OData.EF6/Properties/launchSettings.json

```diff
@@ -12,7 +12,7 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "opstenant",
+      "launchUrl": "opstenant/withoutenablequery?$top=5",
```

Comment:
Shouldn't have to change - see the comment on StartUp.cs.

Reply:
I missed that comment, sorry for that. I changed to get it in line with the `WebAPI.OData.EFCore`-settings. If it makes more sense I revert this change

Reply:
Change reverted